addons/invtracker/invtracker.lua

```diff
@@ -0,0 +1,460 @@
+defaults.SlotImage.status.tempItem.background.color.green = 0
+defaults.SlotImage.status.tempItem.background.color.blue = 0
+defaults.SlotImage.status.empty = {}
+defaults.SlotImage.status.empty.color = {}
+defaults.SlotImage.status.empty.color.alpha = 150
+defaults.SlotImage.status.empty.color.red = 0
+defaults.SlotImage.status.empty.color.green = 0
+defaults.SlotImage.status.empty.color.blue = 0
+defaults.SlotImage.status.empty.background = {}
+defaults.SlotImage.status.empty.background.color = {}
+defaults.SlotImage.status.empty.background.color.alpha = 150
+defaults.SlotImage.status.empty.background.color.red = 0
+defaults.SlotImage.status.empty.background.color.green = 0
+defaults.SlotImage.status.empty.background.color.blue = 0
+
+local settings = config.load(defaults)
+config.save(settings)
+
+settings.SlotImage.equipment.index = 1
+settings.SlotImage.inventory.index = 2
+settings.SlotImage.mogSafe.index = 3
+settings.SlotImage.mogStorage.index = 4
+settings.SlotImage.mogLocker.index = 5
+settings.SlotImage.mogSatchel.index = 6
+settings.SlotImage.mogSack.index = 7
+settings.SlotImage.mogCase.index = 8
+settings.SlotImage.mogWardrobe.index = 9
+settings.SlotImage.tempInventory.index = 10
+settings.SlotImage.box = {}
+settings.SlotImage.box.texture = {}
+settings.SlotImage.box.texture.path = windower.addon_path..'slot.png'
+settings.SlotImage.box.texture.fit = true
+settings.SlotImage.box.size = {}
+settings.SlotImage.box.size.height = 2
+settings.SlotImage.box.size.width = 2
+settings.SlotImage.box.repeatable = {}
+settings.SlotImage.box.repeatable.x = 1
+settings.SlotImage.box.repeatable.y = 1
+settings.SlotImage.background = {}
+settings.SlotImage.background.texture = {}
+settings.SlotImage.background.texture.path = windower.addon_path..'slot.png'
+settings.SlotImage.background.texture.fit = true
+settings.SlotImage.background.size = {}
+settings.SlotImage.background.size.height = 3
+settings.SlotImage.background.size.width = 3
+settings.SlotImage.background.repeatable = {}
+settings.SlotImage.background.repeatable.x = 1
+settings.SlotImage.background.repeatable.y = 1
+
+local yRes = windower.get_windower_settings().ui_y_res
+local xRes = windower.get_windower_settings().ui_x_res
+local xBase = settings.SlotImage.pos.x
+local yBase = settings.SlotImage.pos.y
+local current_block = 0
+local current_slot = 1
+local current_row = 1
+local current_column = 1
+local last_column = 1
+local items = {}
+local slot_images = {}
+local update_time = 0
+
+windower.register_event('load',function()
+    if windower.ffxi.get_info().logged_in then
+        initialize()
+    end
+end)
+
+windower.register_event('login', function()
+    initialize()
+    update_time = os.time()
+end)
+
+windower.register_event('logout', function(...)
+    hide()
+    clear()
+end)
+
+windower.register_event('incoming chunk',function(id,org,modi,is_injected,is_blocked)
+    if (os.time() - update_time > 30 and id == 0x50 or id == 0xE0 or id == 0x1D) then
```

Comment:
The `update_time` variable is never updated. I assume the idea is to prevent spamming, so you only update once every x ms?

Reply:
It's only intented to work when logging in, since you get the incoming chunk event several times per second and really lags the game. With this I validate that 30 seconds have elapsed before actually drawing the grid.

I've changed the name to start_time instead of update_time to reflect this.

Reply:
You get the same packet every time you zone - see my suggestion below about packet `0x00A`. As far as FFXI is concerned, zoning is no different from logging in, from a packet standpoint.